### ISSUE TITLE
SSR compilation issue on iView and bootstrap-vue

### ISSUE THREAD

**Issue by zelid** [devCrossNet/vue-starter#212]
I've tried to install iView and bootstrap-vue components with vue-starter one by one according to instructions on their websites.

Everything works fine until the first time SSR happens. Looks like the issue is related to vue-starter because few Vue libraries fails the same way.

This is the error:
`[1] error: error during rendering: /shop; error: {"stack":"/Users/alex/Workspace/kmb-ts/vue-starter-master/node_modules/iview/dist/styles/iview.css:1\n(function (exports, require, module, __filename, __dirname) { .ivu-load-loop{-webkit-animation:ani-load-loop 1s linear infinite;animation:ani-load-loop 1s linear infinite}@-webkit-keyframes ani-load-loop{from{-webkit-transform:rotate(0);transform:rotate(0)}50%{-webkit-transform:rotate(180deg);transform:rotate(180deg)}to{-webkit-transform:rotate(360deg);transform:rotate(360deg)}}@keyframes ani-load-loop{from{-webkit-transform:rotate(0);transform:rotate(0)}50%{-webkit-transform:rotate(180deg);transform:rotate(180deg)}to{-webkit-transform:rotate(360deg);transform:rotate(360deg)}}.input-group-error-append,.input-group-error-prepend{background-color:#fff;border:1px solid #ed4014}.input-group-error-append .ivu-select-selection,.input-group-error-prepend .ivu-select-selection{background-color:inherit;border:1px solid transparent}.input-group-error-prepend{border-right:0}.input-group-error-append{border-left:0}.ivu-breadcr\n\nSyntaxError: Unexpected token .\n    at new Script (vm.js:74:7)\n    at createScript (vm.js:246:10)\n    at Object.runInThisContext (vm.js:298:10)\n    at Module._compile (internal/modules/cjs/loader.js:657:28)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)\n    at Module.load (internal/modules/cjs/loader.js:599:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:530:3)\n    at Module.require (internal/modules/cjs/loader.js:637:17)\n    at require (internal/modules/cjs/helpers.js:20:18)\n    at r (/Users/alex/Workspace/kmb-ts/vue-starter-master/node_modules/vue-server-renderer/build.js:8335:16)\n    at eval (webpack-internal:///iview/dist/styles/iview.css:1:18)\n    at Object.iview/dist/styles/iview.css (isomorphic.js:626:1)\n    at __webpack_require__ (isomorphic.js:27:30)\n    at eval (webpack-internal:///./src/app/app.ts:16:85)\n    at Module../src/app/app.ts (isomorphic.js:224:1)","message":"Unexpected token ."}`

Basically whenever `.css` file is added to `.ts` (in my case `import 'iview/dist/styles/iview.css';` to `app.ts`) file from within the node_packages folder the error happens.

Any ideas where to look in order to solve it?
Could be an integration issue related to webpack 4, it's plugins, TypeScript or isomorphic.ts

**Comment by devCrossNet** [comment]
Hey @zelid thanks for reporting this.

I will try to reproduce it later this week and come back to you. My first guess is that you have to add a webpack-loader for CSS files in the webpack base config. because the vue-starter is just set up for SCSS/SASS.

You can try it by adding one in the meantime and see if it works.

**Comment by devCrossNet** [comment]
@zelid see the new documentation section: https://vue-starter.herokuapp.com/docs/guide/third-party.html

this should help you even if it's a guide for vuetify.
